Mesh_3/include/CGAL/Mesh_3/Mesher_3.h

```diff
@@ -719,6 +720,20 @@ initialize()
   else
 #endif // CGAL_LINKED_WITH_TBB
   {
+
+    if (r_c3t3_.number_of_facets() == 0)
+    {
+      CGAL::warning_fail("c3t3.number_of_facets() == 0",
+        "CGAL/Mesh_3/Mesher_3.h",
+        725,
+        "Warning : The mesh refinement process can't start.\n"
```

Comment:
We do not want to hard-code the file and line, but use the macros `__FILE__` and `__LINE__`.
```suggestion
      CGAL::warning_fail("c3t3.number_of_facets() == 0",
        __FILE__,
        __LINE__,
        "Warning : The mesh refinement process can't start.\n"
```
Those macros are [ones of the few officially documented in the C++ norm](https://en.cppreference.com/w/cpp/preprocessor/replace#Predefined_macros).